src/src/utils/noir/honk_transcript.cairo

```diff
@@ -191,14 +191,14 @@ struct HonkTranscript {
 #[generate_trait]
 impl HonkTranscriptImpl of HonkTranscriptTrait {
     fn from_proof<T, impl Hasher: IHasher<T>, impl Drop: Drop<T>>(
-        honk_proof: HonkProof,
+        honk_vk: @HonkVk, honk_proof: HonkProof,
```

Comment:
Please only pass the relevant fields of HonkVk in the signature instead of the full Vk struct. 
It will save steps in Cairo as for non inlined function calls, it costs as many cairo steps as the size (in field elements) of all the arguments of the function to call the function. 


ie : pass circuit_size, publicinputs_size and offset as arguments of this function instead of HonkVk object